app/javascript/src/Users/utils/index.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import type { Feature } from 'Users/types'
+
+const FEATURE_NAMES: { [feature: Feature]: string } = {
+  portal: 'Developer Portal',
+  finance: 'Billing',
+  settings: 'Settings',
+  partners: 'Developer Accounts -- Applications',
+  monitoring: 'Analytics',
+  plans: 'Integration & Application Plans',
+  policy_registry: 'Policy Registry'
+}
+
+const FEATURES_GRANTING_SERVICE_ACCESS = ['partners', 'monitoring', 'plans']
+
+export function getFeatureName (feature: Feature): string {
+  if (feature in FEATURE_NAMES) {
+    return FEATURE_NAMES[feature]
+  }
+
+  throw new Error(`${feature} is not a known feature`)
+}
+
+export function isServicePermissionsGranted (features: Feature | 'services' | Array<Feature | 'services'>): boolean {
```

Comment:
yeah, It's a bit weird how the string `services` mingles with Features, maybe define a type for that string?